frontend/public/components/utils/dropdown.jsx

```diff
@@ -326,7 +326,7 @@ export class Dropdown extends DropdownMixin {
               {titlePrefix && `${titlePrefix}: `}
               <span className="dropdown__not-btn__title">{title}</span>&nbsp;<Caret />
             </div>
-            : <button aria-haspopup="true" onClick={this.toggle} onKeyDown={this.onKeyDown} type="button" className={classNames('btn', 'btn-dropdown', 'dropdown-toggle', buttonClassName ? buttonClassName : 'btn-default')} id={this.props.id}>
+            : <button aria-haspopup="true" onClick={this.toggle} onKeyDown={this.onKeyDown} type="button" className={classNames('btn', 'btn-dropdown', 'dropdown-toggle', buttonClassName ? buttonClassName : 'btn-default')} id={this.props.id} {...this.props.id && {'aria-describedby' : `${this.props.id}-help`}}>
```

Comment:
We should explicitly pass in a described by prop and not assume some naming convention based on ID. This also lets us leave the attribute off when there's no describedby block instead of having an invalid value.